build.gradle

```diff
@@ -174,8 +174,6 @@ task gems(type: Exec) {
     outputs.dir file('src/dist/gems/vendor/')
     workingDir 'src/dist/gems'
     executable 'sh'
-    // HACK: Workaround for bundler/bundler#4494
-    args '-c', 'bundle _1.11.2_ package --no-install'
```

Comment:
This can't be removed entirely, it has to be reverted to the version prior to 4c43460d8e45e3b263b9bdf8148ce12c4b215027.